Resources/Locale/en-US/objectives/conditions/steal-condition.ftl

```diff
@@ -1,2 +1,5 @@
-objective-condition-steal-title = Steal {$amount}{$itemName}.
+objective-condition-steal-title = Steal the {$owner}'s {$itemName}.
 objective-condition-steal-description = We need you to steal {$itemName}. Don't get caught.
+
+objective-condition-steal-station = station
+objective-condition-steal-Ian = head of personnel's corgi
```

Comment:
```suggestion
objective-condition-steal-Ian = head of personnel's corgi's
```
i think?

Reply:
![image](https://user-images.githubusercontent.com/60792108/178169880-7b5ff973-d587-441b-baae-8cd894aee646.png)
Steal the head of personnel's corgi's's prime cut corgi meat?